node/src/components/consensus/era_supervisor.rs

```diff
@@ -468,6 +469,7 @@ impl EraSupervisor {
             start_time,
             seed,
             now,
+            self.chainspec.highway_config.compute_rewards,
```

Comment:
Is this necessary? Can't we just configure `round_seigniorage_rate = [0, 1]` instead?

Reply:
Alternatively, yes, but auction logic would have to go with the complete flow as the round seigniorage rate is known to auction only. By passing rewards = 0 from the era supervisor, we can exit early inside the auction flow.

Reply:
OK; then we could make it independent of Highway, and just set the rewards to 0 [in the era supervisor](https://github.com/casper-network/casper-node/blob/9e75f2b8eb353ccd89617141cdb557ff16fa9f9d/node/src/components/consensus/era_supervisor.rs#L920), regardless of what the consensus protocol returns? (Unless we are worried about the cost of computing the rewards inside Highway.)

(Not a blocker; just a thought.)